docs/creators/configure-forem.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuring Forem
+---
+
+#Basic Site Configuration Guide for Forem
+
+##Overview
+
+As a Forem admin, one of the first steps of managing your site will be to tailor your content, branding and other important details based on your community goals. This is made simple by the configuration page which can be found at https://<span></span>your.forem.url/internal/config.
+
+_We advise that you first complete the minimum set up before sending out your Forem link to your community._
+
+##Complete your Set Up, Configure your Forem
+
+Once your Forem instance is set up for the first time, you will most likely see the following banner:
+
+![Banner showing outstanding site configuration](https://dev-to-uploads.s3.amazonaws.com/i/2nosvfr7l47ymipmyh4o.png)
+
+This banner indicates that the minimum fields to configure your Forem have not been met.
```

Comment:
What about `This banner indicates that the Forem setup process hasn't been completed yet` ?

Reply:
✅